maybe_rayon/src/lib.rs

```diff
@@ -1,8 +1,6 @@
-#[cfg(not(feature = "parallel"))]
-use core::{
-    iter::{FlatMap, IntoIterator, Iterator},
-    slice::{Chunks, ChunksExact, ChunksExactMut, ChunksMut},
-};
+#![cfg_attr(not(std), no_std)]
+
+extern crate alloc;
```

Comment:
We only need alloc conditionally imported here
```suggestion
#[cfg(not(feature = "parallel"))]
extern crate alloc;
```